### ISSUE TITLE
Unknown command spam in console

### ISSUE THREAD

**Issue by bloggy** [bensku/Skript#118]
Whenever I execute a console command (via console) I get the "unknown command" warning in the console which spams my server.

This is the script command I am executing:
http://hastebin.com/uheqifowuv.vbs

I am on spigot 1.9.4.

**Comment by bensku** [comment]
Does using /skript:<your command name> work?

**Comment by bloggy** [comment]
Yes, it works and there is no console spam with /skript: before the command. But that is not a good solution. How can this be fixed?

**Comment by bensku** [comment]
Could it be that some other plugin has command with exactly same name? I.e. could you try if that command exists when you're running server without that script?

**Comment by TheBentoBox** [comment]
I can confirm that I've always had this issue (since long before this fork, but it persists in the fork). Running commands from console tends to work just fine, but it also sends an unknown command message, even if the command isn't registered in a different plugin. e.g. here's me running my Skripted /playtime command on a player

```
12.07 09:45:29 [Server] INFO Faelisa's total time online is 5 day(s) 4 hour(s) 35 minute(s) 23 second(s).
12.07 09:45:29 [Server] INFO Unknown command. Type "/help" for help.
```

The `Unknown command` shows up after the command result, even though it worked just fine.

**Comment by Pilvinen** [comment]
Yes, I can confirm, I also have this. It's been around for a very long time. Doesn't really break any functionality, just an oddity.

**Comment by Nicofisi** [comment]
This has been a known bug since the beginning of Skript, adn as far as I know Njol finally said that this won't be fixed (by him).
https://github.com/Njol/Skript/issues/194

**Comment by xXAndrew28Xx** [comment]
I just saw this. https://www.spigotmc.org/threads/small-easy-register-command-without-plugin-yml.38036/ It uses NMS but could easily be changed to use Reflection to access NMS (for cross-version compatability)

**Comment by bloggy** [comment]
This still exists, any solutions yet? At least if we could stop the console spam^^

**Comment by bensku** [comment]
You can use logfilters to get rid of messages. And msybe I will fix this at some point, but low priority.

**Comment by Adrihun** [comment]
I'm not good at java and coding. So i can't go into the code and update it, etc... If anyone can fix this, please fix it and upload the newer fixed, jar! Thanks 👍

**Comment by Adrihun** [comment]
Why closed?

**Comment by xXAndrew28Xx** [comment]
Its not closed..

**Comment by Adrihun** [comment]
Please fix this asap.. Please?

**Comment by TheBentoBox** [comment]
Why ASAP ...? It's not really a huge issue, just a minor annoyance. There are far more important bugs to worry about than this one.

**Comment by Adrihun** [comment]
Important bugs?
Show me xd

**Comment by TheBentoBox** [comment]
#485, or #483, or #482? Those are just from the latest release alone.

**Comment by Snow-Pyon** [comment]
This can be closed once #1013 is merged. Also, for 1.9.0 users, make sure you have the latest build made for that version, such as there is a specific build where ServerCommandEvent#setCancelled is ignored, and that's what my PR uses to fix this.

By the way, it's kinda hilarious how the such easily fixeable bug was actually fixed on #77 but due to how Skript handled command execution, which was executing the command separatedly and then setting the actual executed command to "", the error persisted.

Oh well, I think the 5 hours I spent finding the root cause of this weren't for nothing.

**Comment by SlimeDog** [comment]
Thanks. I've been using a console filter for years to elide that message. All better now.
